stellar_account_prometheus_exporter/exporter.py

```diff
@@ -87,18 +92,18 @@ def do_GET(self):
                     self.error(504, 'Error retrieving data from {}'.format(url))
                     return
                 if not r.ok:
-                    self.error(504, 'Error retrieving data from {}'.format(url))
-                    return
+                    m_account_exists.labels(*account_labels).set(0)
+                    continue
```

Comment:
Nice and clean!
I think its worth doing the same thing for two error conditions below (missing `balances` and `subentry_count`
If there is an easy to log those error to console that would be a nice bonus but definitely don't block on it

Reply:
I added metrics to denote error scenarios for the missing `balances` and `subentry_count` along with logger.